docs/ADL2/master07.10-adl_definition.adoc

```diff
@@ -2,20 +2,88 @@
 
 The `definition` section contains the main formal definition of the archetype, and is written in the Constraint Definition Language (cADL). A typical `definition` section is as follows:
 
+[tabs,sync-group-id=adl-example]
+====
+at-coded ADL2::
++
+[source, adl]
+--------
+definition
+    OBSERVATION[at0000] ∈ {                                              -- blood pressure measurement
+        name ∈ {                                                         -- any synonym of BP
+            DV_CODED_TEXT[at9000] ∈ {
+                defining_code ∈ {
+                    CODE_PHRASE[at9001] ∈ {[ac1]}
```

Comment:
Lots of ac codes in this at example.

Reply:
https://github.com/openEHR/specifications-AM/pull/16#discussion_r1670057543